src/pages/StartPage.jsx

```diff
@@ -12,48 +16,84 @@ export const StartPage = ({ onSetupComplete }) => {
   //The start page here will set initial data to the store,
   //used then for the app itself
   const settingsState = useSelector(state => state.settings);
-  const currentDayStateMood = useSelector(state => state.mood);
-  const [name, setName] = useState('');
-  const [mood, setMood] = useState('');
+  const [name, setName] = useState(settingsState.name);
+  const [focusTimerLength, setFocusTimerLength] = useState(
+    settingsState.focusTimerLengthMS / (60 * 1000) // convert milliseconds to minutes
+  );
+  const [breatheTimerLength, setBreatheTimerLength] = useState(
+    settingsState.breatheTimerLengthMS / (60 * 1000) // convert milliseconds to minutes
+  );
 
   const dispatch = useDispatch();
 
+  useEffect(() => {
+    //Check localStorage for existing data
+    //I think this could be refactored better?
+    const storedSettings = JSON.parse(localStorage.getItem('settings'));
+    const storedName = storedSettings ? storedSettings.name : null;
+
+    //console.log(storedName);
+
+    if (storedName) {
+      // If data is found, skip the StartPage
+      onSetupComplete();
+    }
+  }, [onSetupComplete]);
```

Comment:
does this need to be checked when we load the app? This might need to be an empty array then on line 41

Reply:
right, let's remove that dependency